javascriptv3/example_code/bedrock-runtime/hello.js

```diff
@@ -0,0 +1,79 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+/**
+ * @typedef {Object} Content
+ * @property {string} text
+ *
+ * @typedef {Object} Usage
+ * @property {number} input_tokens
+ * @property {number} oputput_tokens
+ *
+ * @typedef {Object} ResponseBody
+ * @property {Content[]} content
+ * @property {Usage} usage
+ */
+
+import { fileURLToPath } from "url";
+import { defaultProvider } from "@aws-sdk/credential-provider-node";
+import {
+  BedrockRuntimeClient,
+  InvokeModelCommand,
+} from "@aws-sdk/client-bedrock-runtime";
+
+const AWS_REGION = "us-east-1";
+
+const MODEL_ID = "anthropic.claude-3-haiku-20240307-v1:0";
+const PROMPT = "Hi. In a short paragraph, explain what you can do.";
+
+const hello = async () => {
+  console.log("=".repeat(35));
```

Comment:
You should check out `libs/scenario/scenario-example.js`. I use it for orchestrating multi step examples.